src/otserv.cpp

```diff
@@ -63,6 +64,19 @@ std::mutex g_loaderLock;
 std::condition_variable g_loaderSignal;
 std::unique_lock<std::mutex> g_loaderUniqueLock(g_loaderLock);
 
+/**
+ * It is preferable to keep the close button off as it closes the server without saving  (within five seconds)
+ * Choose to use "CTROL + C" or "CTROL + BREAK" for security close
+ * To activate/desactivate window
+ * \param MF_GRAYED Disable the "x" (force close) button
+ * \param MF_ENABLED Enable the "x" (force close) button
+*/
+void closeButton() {
```

Comment:
This function needs a better name: "disableCloseButton" maybe.

Reply:
I didn't add it, because in case of changing it to "enabled", it wouldn't make sense. But, with the idea of sending it to config.lua, then it becomes viable, I'll do it.